src/sinks/datadog/mod.rs

```diff
@@ -63,12 +62,14 @@ pub enum DatadogApiError {
     Forbidden,
 }
 
+pub type BodyBox = http_body::combinators::BoxBody<hyper::body::Bytes, hyper::Error>;
+
```

Comment:
The best bet would be to import the type alias from our `http` module, i.e. `use crate::http::BoxBody;`